lib/EmitVerilog/EmitVerilog.cpp

```diff
@@ -864,6 +864,24 @@ class ExprEmitter : public ExprVisitor<ExprEmitter, SubExprInfo>,
   SubExprInfo visitComb(rtl::ConcatOp op);
   SubExprInfo visitComb(rtl::ExtractOp op);
 
+  // RTL Comparison Operations
+  SubExprInfo visitComb(rtl::LTOp op) {
+    return emitSignedBinary(op, Comparison, "<");
+  }
+  SubExprInfo visitComb(rtl::LEQOp op) {
+    return emitSignedBinary(op, Comparison, "<=");
```

Comment:
Yeah as you noticed, this isn't going to be enough.  `emitSignedBinary` is poorly named (my fault), it really emits operands respecting the signedness of the types of operands (`emitBinaryRespectingSigns`??).  This won't work for the RTL dialect, as the standard integer types don't carry signs.  We will need another lowering path here.

As you mention, it is probably also broken for div and mod, so this can be a follow up fix.  Please file a bug to fix this up, thanks for noticing this!

Reply:
Filed: #139 
I was thinking a tblgen bit marking signed ops would be useful.

Reply:
I suspect that EmitVerilog will get a significant refactor at some point: we can split it out to use OpInterfaces, allowing decentralized extension of EmitVerilog, instead of there being one file that knows all the supported ops.  We shouldn't tackle this until all the FIRRTL stuff is removed though IMO.